LethalAPI.Core/Events/Events.cs

```diff
@@ -0,0 +1,114 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="Events.cs" company="LethalAPI Modding Community">
+// Copyright (c) LethalAPI Modding Community. All rights reserved.
+// Licensed under the GPL-3.0 license.
+// </copyright>
+// Taken from EXILED (https://github.com/Exiled-Team/EXILED)
+// Licensed under the CC BY SA 3 license. View it here:
+// https://github.com/Exiled-Team/EXILED/blob/master/LICENSE.md
+// Changes: Namespace adjustments, and potential removed properties.
+// -----------------------------------------------------------------------
+
+namespace LethalAPI.Core.Events;
+
+using System;
+using System.Diagnostics;
+
+using Features;
+using UnityEngine.SceneManagement;
+
+/// <summary>
+/// The main events api for patching and un-patching methods.
+/// </summary>
+public sealed class Events
+{
+#pragma warning disable SA1401 // literally how should this be private when to other instances outside this class use it???????
+    /// <summary>
+    /// Indicates whether debug patches and debug patch logs should be enabled.
+    /// </summary>
+    internal const bool DebugPatches = true;
```

Comment:
Shouldn't debug features be disabled by default?

Reply:
I thought I disabled it before making the pr. I can disable it again. Ultimately the option wants to move to a dedicated config.